sdk/keyvault/azkeys/client.go

```diff
@@ -72,7 +72,7 @@ type CreateKeyOptions struct {
 	Curve *KeyCurveName `json:"crv,omitempty"`
 
 	// The attributes of a key managed by the key vault service.
-	KeyAttributes *KeyAttributes  `json:"attributes,omitempty"`
+	KeyProperties *KeyProperties  `json:"attributes,omitempty"`
```

Comment:
Do we need to repeat Key everywhere? Can't the field names be just Properties and Operations?
Where else can we apply this shortening?

Reply:
These were consistent with every other language, not to mention fairly consistent with the underlying data types common for a JWT like "kid" for "KeyID", "key_ops" for "KeyOperations", etc. I understand that language idiomaticity is primary, but is there any great reason to differ from other languages?

Reply:
Go tries very hard not to repeat names which they call stuttering. So, repeating "Key" falls under this category. You can get a sense of how much there is written about by doing this [search](https://www.google.com/search?q=golang+stuttering). So, for Go, we want to reduce this as much as possible as it is what the community expects.

Reply:
To rant just a little, the verbosity of .NET bleeds into everything.  This is a prime example.  As a Go developer, it's somewhat frustrating that we get saddled with patterns from tier 1 languages that clash with our idioms.